src/system.cpp

```diff
@@ -632,21 +627,20 @@ int System::JustifyY(FunctorParams *functorParams)
     JustifyYParams *params = dynamic_cast<JustifyYParams *>(functorParams);
     assert(params);
 
-    bool systemOnly = params->m_doc->GetOptions()->m_justifySystemsOnly.GetValue();
+    const double systemJustificationFactor = params->m_doc->GetOptions()->m_justificationSystem.GetValue();
+    const double shift = systemJustificationFactor / params->m_justificationSum * params->m_spaceToDistribute;
```

Comment:
That would, a rear case. It means that all factors should be set to 0 and justification enabled.
In any case, I'll add a check in Page::JustifyVertically to avoid reaching here with zeroes.

Reply:
added check in Page::JustifyVertically